Dockerfile

```diff
@@ -9,7 +9,7 @@ VOLUME ["/var/lib/odoo"]
 EXPOSE 8069 8072
 
 # Subimage triggers
-ONBUILD ENTRYPOINT ["/opt/odoo/common/entrypoint.sh"]
+ONBUILD ENTRYPOINT ["/opt/odoo/common/entrypoint"]
```

Comment:
Please call it `entrypoint.py`. Having to avoid file extensions with current `run-parts` implementation sucks. This also leads to removing [the corresponding FAQ](https://github.com/Tecnativa/docker-odoo-base#why-my-99-whateversh-script-in-optodood-does-not-execute).